test/functional/feature_logging.py

```diff
@@ -15,21 +15,25 @@ def set_test_params(self):
         self.num_nodes = 1
         self.setup_clean_chain = True
 
+    def relative_log_path(self, name):
+        return os.path.join(self.nodes[0].datadir, "regtest", name)
```

Comment:
nit: IIRC `datadir` is absolute, so the `relative` in the method name is confusing.